posthog/hogql/functions/mapping.py

```diff
@@ -484,6 +484,7 @@ class HogQLFunctionMeta:
     "JSONLength": HogQLFunctionMeta("JSONLength", 1, None),
     "JSONArrayLength": HogQLFunctionMeta("JSONArrayLength", 1, None),
     "JSONType": HogQLFunctionMeta("JSONType", 1, None),
+    "JSONExtract": HogQLFunctionMeta("JSONExtract", 1, None),
```

Comment:
Do you have an example of usage of this function? The key thing is that it requires a second argument:
```suggestion
    "JSONExtract": HogQLFunctionMeta("JSONExtract", 2, None),
```
According to the CH docs, `JSONExtract(foo, <type>)` is just alternative syntax for `JSONExtract<type>(foo)`, so I'm not sure if it'd actually be helpful